commands/import_test.go

```diff
@@ -0,0 +1,209 @@
+package commands
+
+import (
+	"testing"
+	"fmt"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/activecm/rita/config"
+)
+
+func TestSetRolling(t *testing.T) {
+	type cfg = config.RollingStaticCfg // including the definition here for reference:
+	// 	DefaultChunks int `yaml:"DefaultChunks" default:"12"`
+	// 	Rolling       bool
+	// 	CurrentChunk  int
+	// 	TotalChunks   int
+
+	type tc struct{
+		msg string
+		dbExists bool
+		dbIsRolling bool
+		dbCurrChunk int
+		dbTotalChunks int
+		userIsRolling bool
+		userCurrChunk int
+		userTotalChunks int
+		cfgDefaultChunks int
+		expected cfg
+		err bool
+	}
+
+	// this is the sentinel value that signifies that a user did not supply
+	// a command line value for --chunk or --numchunks
+	const blank int = -1
+	// these are used to help make the test table below (a little) more readable
+	const exists bool = true
+	const rolling bool = true
+	const returnsError bool = true
+	const default12 int = 12
+	const default24 int = 24
+
+	testCases := []tc{
```

Comment:
I tried to balance being concise with readable. I know it's not great but I think it's better than using 10 lines for each test case to fill out the `tc` struct the long way.  I'm more than happy to explain what's going on here if anyone has questions.

Please feel free to suggest test cases you'd like to see that I missed.